test/test_framework.py

```diff
@@ -554,10 +583,6 @@ class CustomObject:
 
         self.assertEqual(dict(obj.state.dict), {"a": {"b": "c"}})
 
-        self.assertEqual(obj.changes, 5)
```

Comment:
This is the most of the point of this test. If you drop the assertion, the test becomes somewhat irrelevant. We need something to verify here on its place, or at least make sure that the code we have in the test is still verifying something relevant for us.